Stratis.Bitcoin/RPC/RPCFeature.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Stratis.Bitcoin.Builder;
+using Stratis.Bitcoin.Builder.Feature;
+using Stratis.Bitcoin.Configuration;
+using Stratis.Bitcoin.Utilities;
+using Stratis.Bitcoin.Logging;
+using Microsoft.Extensions.Logging;
+using System;
+
+namespace Stratis.Bitcoin.RPC
+{
+    public class RPCFeature : FullNodeFeature
+    {
+        private readonly FullNode fullNode;
+        private readonly NodeSettings nodeSettings;
+        public RPCFeature(FullNode fullNode, NodeSettings nodeSettings)
+        {
+            this.fullNode = fullNode;
+            this.nodeSettings = Guard.NotNull(nodeSettings, nameof(nodeSettings));
+        }
+
+        public override void Start()
+        {
+            if (this.nodeSettings.RPC != null)
```

Comment:
This is probably not the best place to check if RPC is set. 
If the start method is called on this feature its likely the implementer had the intention to enable RPC - builder.UseRPC()

I'd say at this point if RPC is null throw a NodeBuilderException